app/code/Magento/Cron/etc/db_schema_whitelist.json

```diff
@@ -13,7 +13,8 @@
         "index": {
             "CRON_SCHEDULE_JOB_CODE": true,
             "CRON_SCHEDULE_SCHEDULED_AT_STATUS": true,
-            "CRON_SCHEDULE_SCHEDULE_ID_STATUS": true
+            "CRON_SCHEDULE_SCHEDULE_ID_STATUS": true,
```

Comment:
@ihor-sviziev Thanks again for working through this.

Did it need this addition for tests? It's strange as in the db_schema.xml I reverted that index addition, it's not there in this PR anymore. Seems strange - maybe the test runner isn't seeing the removal properly?

Reply:
Basically it was needed for fixing SVC failure.
In general this file contrails the list of indexes that it could manage. So as we don't have this index in the db_schema.xml anymore - it will be automatically removed. In case if it wasn't present in your DB - it will not be created also.